### DISCUSSION TITLE
Webserver access

### DISCUSSION THREAD

**Discussion by Trev114** [PlummersSoftwareLLC/NightDriverStrip#156] [Q&A]
I tried to build the webserver image then upload I keep getting an error message that JS can't be uploaded. How do I go about fixing this?

**Comment by wsirons** [comment]
I'm also trying to get the webserver to work.  I've found that the SPIFFS partition is too small to hold the js file and since I'm not using OTA yet, have been trying to change the partition sizes in paritions_custom.csv.  I have been able to to upload the filesystem to the ESP32, but then get stuck with [AsyncTCP.cpp:1251] begin(): failed to start task during startup and cannot connect to the IP address.
